neuro_san/service/agent_servicer_to_server.py

```diff
@@ -30,20 +26,17 @@ class AgentServicerToServer:
     by the same server with a simple addition of an agent name in the gRPC path.
     """
 
-    def __init__(self, servicer: AgentServiceServicer,
-                 agent_name: str = ""):
+    def __init__(self, servicer: AgentServiceServicer):
         """
         Constructor
         """
         self.servicer: AgentServiceServicer = servicer
-        self.agent_name: str = agent_name
 
-    def add_rpc_handlers(self, server: Server):
+    def build_rpc_handlers(self):
         """
-        Adds the RpcMethodHandlers to the server
```

Comment:
We don't add gRPC handlers here anymore, we only build collection of RpcMethodHandlers
to be given to DynamicAgentRouter.

Reply:
Brave! 
I don't recall off the top of my head, but it's possible the "ServicerToServer" nomenclature means something very specific in gRPC land and if so with the interface change here it might warrant a rename.